src/Read_Forcing.f90

```diff
@@ -13,77 +13,75 @@ SUBROUTINE Read_Forcing
 no_flow=0
 no_heat=0
 do nr=1,nreach
-  do nc=1,no_cells(nr)-1 
-    no_flow=no_flow+1
-    no_heat=no_heat+1
-!
```

Comment:
NREC_FLOW and NREC_HEAT are no longer needed since they were used for files created as DIRECT ACCESS
JRY 1/20